### ISSUE TITLE
invalid syntax when process data

### ISSUE THREAD

**Issue by mc171819** [open-mmlab/mmdetection3d#1602]
When I run command in the picture below, I meet an error like this:
![image](https://user-images.githubusercontent.com/81420309/176991339-4749693e-4cf6-4d47-bea1-85123ed188ef.png)
This is the env I use, I wonder what's the problem
# packages in environment at /home/mc/.conda/envs/mmlab:
#
# Name                    Version                   Build  Channel
absl-py                   1.1.0                     <pip>
addict                    2.4.0                     <pip>
argon2-cffi               21.3.0                    <pip>
argon2-cffi-bindings      21.2.0                    <pip>
attrs                     21.4.0                    <pip>
backcall                  0.2.0                     <pip>
beautifulsoup4            4.11.1                    <pip>
black                     22.6.0                    <pip>
blas                      1.0                         mkl
bleach                    5.0.1                     <pip>
bzip2                     1.0.8                h7b6447c_0
ca-certificates           2022.4.26            h06a4308_0
cachetools                5.2.0                     <pip>
certifi                   2022.6.15        py37h06a4308_0
cffi                      1.15.1                    <pip>
charset-normalizer        2.1.0                     <pip>
click                     8.1.3                     <pip>
cudatoolkit               11.3.1               h2bc3f7f_2
cycler                    0.11.0                    <pip>
debugpy                   1.6.0                     <pip>
decorator                 5.1.1                     <pip>
defusedxml                0.7.1                     <pip>
descartes                 1.1.0                     <pip>
entrypoints               0.4                       <pip>
fastjsonschema            2.15.3                    <pip>
ffmpeg                    4.3                  hf484d3e_0    pytorch
fire                      0.4.0                     <pip>
flake8                    4.0.1                     <pip>
fonttools                 4.33.3                    <pip>
freetype                  2.11.0               h70c0345_0
giflib                    5.2.1                h7b6447c_0
gmp                       6.2.1                h295c915_3
gnutls                    3.6.15               he1e5248_0
google-auth               2.9.0                     <pip>
google-auth-oauthlib      0.4.6                     <pip>
grpcio                    1.47.0                    <pip>
idna                      3.3                       <pip>
imageio                   2.19.3                    <pip>
importlib-metadata        4.2.0                     <pip>
importlib-resources       5.8.0                     <pip>
iniconfig                 1.1.1                     <pip>
intel-openmp              2021.4.0          h06a4308_3561
ipykernel                 6.15.0                    <pip>
ipython                   7.34.0                    <pip>
ipython-genutils          0.2.0                     <pip>
ipywidgets                7.7.1                     <pip>
jedi                      0.18.1                    <pip>
Jinja2                    3.1.2                     <pip>
joblib                    1.1.0                     <pip>
jpeg                      9e                   h7f8727e_0
jsonschema                4.6.1                     <pip>
jupyter                   1.0.0                     <pip>
jupyter-client            7.3.4                     <pip>
jupyter-console           6.4.4                     <pip>
jupyter-core              4.10.0                    <pip>
jupyterlab-pygments       0.2.2                     <pip>
jupyterlab-widgets        1.1.1                     <pip>
kiwisolver                1.4.3                     <pip>
lame                      3.100                h7b6447c_0
lcms2                     2.12                 h3be6417_0
ld_impl_linux-64          2.38                 h1181459_1
libffi                    3.3                  he6710b0_2
libgcc-ng                 11.2.0               h1234567_1
libiconv                  1.16                 h7f8727e_2
libidn2                   2.3.2                h7f8727e_0
libpng                    1.6.37               hbc83047_0
libstdcxx-ng              11.2.0               h1234567_1
libtasn1                  4.16.0               h27cfd23_0
libtiff                   4.2.0                h2818925_1
libunistring              0.9.10               h27cfd23_0
libuv                     1.40.0               h7b6447c_0
libwebp                   1.2.2                h55f646e_0
libwebp-base              1.2.2                h7f8727e_0
llvmlite                  0.36.0                    <pip>
llvmlite                  0.31.0                    <pip>
lyft-dataset-sdk          0.0.8                     <pip>
lz4-c                     1.9.3                h295c915_1
Markdown                  3.3.5                     <pip>
MarkupSafe                2.1.1                     <pip>
matplotlib                3.5.2                     <pip>
matplotlib-inline         0.1.3                     <pip>
mccabe                    0.6.1                     <pip>
mistune                   0.8.4                     <pip>
mkl                       2021.4.0           h06a4308_640
mkl-service               2.4.0            py37h7f8727e_0
mkl_fft                   1.3.1            py37hd3c417c_0
mkl_random                1.2.2            py37h51133e4_0
mmcls                     0.23.1                    <pip>
mmcv                      1.4.0                     <pip>
mmdet                     2.25.0                    <pip>
mmdet3d                   1.0.0rc3                  <pip>
mmsegmentation            0.26.0                    <pip>
mypy-extensions           0.4.3                     <pip>
nbclient                  0.6.6                     <pip>
nbconvert                 6.5.0                     <pip>
nbformat                  5.4.0                     <pip>
ncurses                   6.3                  h7f8727e_2
nest-asyncio              1.5.5                     <pip>
nettle                    3.7.3                hbbd107a_1
networkx                  2.2                       <pip>
notebook                  6.4.12                    <pip>
numba                     0.53.0                    <pip>
numba                     0.48.0                    <pip>
numpy                     1.21.5           py37h6c91a56_3
numpy                     1.21.6                    <pip>
numpy                     1.19.5                    <pip>
numpy-base                1.21.5           py37ha15fc14_3
nuscenes-devkit           1.1.9                     <pip>
oauthlib                  3.2.0                     <pip>
opencv-python             4.6.0.66                  <pip>
openh264                  2.1.1                h4ff587b_0
openssl                   1.1.1p               h5eee18b_0
packaging                 21.3                      <pip>
pandas                    1.3.5                     <pip>
pandocfilters             1.5.0                     <pip>
parso                     0.8.3                     <pip>
pathspec                  0.9.0                     <pip>
pexpect                   4.8.0                     <pip>
pickleshare               0.7.5                     <pip>
Pillow                    9.2.0                     <pip>
pillow                    9.0.1            py37h22f2fdc_0
pip                       21.2.2           py37h06a4308_0
platformdirs              2.5.2                     <pip>
plotly                    5.9.0                     <pip>
pluggy                    1.0.0                     <pip>
plyfile                   0.7.4                     <pip>
prettytable               3.3.0                     <pip>
prometheus-client         0.14.1                    <pip>
prompt-toolkit            3.0.30                    <pip>
protobuf                  3.19.4                    <pip>
psutil                    5.9.1                     <pip>
ptyprocess                0.7.0                     <pip>
py                        1.11.0                    <pip>
pyasn1                    0.4.8                     <pip>
pyasn1-modules            0.2.8                     <pip>
pycocotools               2.0.4                     <pip>
pycodestyle               2.8.0                     <pip>
pycparser                 2.21                      <pip>
pyflakes                  2.4.0                     <pip>
Pygments                  2.12.0                    <pip>
pyparsing                 3.0.9                     <pip>
pyquaternion              0.9.9                     <pip>
pyrsistent                0.18.1                    <pip>
pytest                    7.1.2                     <pip>
python                    3.7.13               h12debd9_0
python-dateutil           2.8.2                     <pip>
pytorch                   1.10.0          py3.7_cuda11.3_cudnn8.2.0_0    pytorch
pytorch-mutex             1.0                        cuda    pytorch
pytz                      2022.1                    <pip>
PyWavelets                1.3.0                     <pip>
PyYAML                    6.0                       <pip>
pyzmq                     23.2.0                    <pip>
qtconsole                 5.3.1                     <pip>
QtPy                      2.1.0                     <pip>
readline                  8.1.2                h7f8727e_1
requests                  2.28.1                    <pip>
requests-oauthlib         1.3.1                     <pip>
rsa                       4.8                       <pip>
scikit-image              0.19.3                    <pip>
scikit-learn              1.0.2                     <pip>
scipy                     1.7.3                     <pip>
Send2Trash                1.8.0                     <pip>
setuptools                61.2.0           py37h06a4308_0
Shapely                   1.8.2                     <pip>
six                       1.16.0             pyhd3eb1b0_1
soupsieve                 2.3.2.post1               <pip>
sqlite                    3.38.5               hc218d9a_0
tenacity                  8.0.1                     <pip>
tensorboard               2.9.1                     <pip>
tensorboard-data-server   0.6.1                     <pip>
tensorboard-plugin-wit    1.8.1                     <pip>
termcolor                 1.1.0                     <pip>
terminado                 0.15.0                    <pip>
terminaltables            3.1.10                    <pip>
threadpoolctl             3.1.0                     <pip>
tifffile                  2022.5.4                  <pip>
tinycss2                  1.1.1                     <pip>
tk                        8.6.12               h1ccaba5_0
tomli                     2.0.1                     <pip>
torchaudio                0.10.0               py37_cu113    pytorch
torchvision               0.11.0               py37_cu113    pytorch
tornado                   6.1                       <pip>
tqdm                      4.64.0                    <pip>
traitlets                 5.3.0                     <pip>
typed-ast                 1.5.4                     <pip>
typing_extensions         4.1.1              pyh06a4308_0
typing_extensions         4.3.0                     <pip>
urllib3                   1.26.9                    <pip>
wcwidth                   0.2.5                     <pip>
webencodings              0.5.1                     <pip>
Werkzeug                  2.1.2                     <pip>
wheel                     0.37.1             pyhd3eb1b0_0
wheel                     0.37.1                    <pip>
widgetsnbextension        3.6.1                     <pip>
xz                        5.2.5                h7f8727e_1
yapf                      0.32.0                    <pip>
zipp                      3.8.0                     <pip>
zlib                      1.2.12               h7f8727e_2
zstd                      1.5.2                ha4553b6_0

**Comment by mc171819** [comment]
@lindahua
